### DISCUSSION TITLE
Idea of automation list with categories

### DISCUSSION THREAD

**Discussion by LeoI5** [home-assistant/frontend#8815] [Automations / scripts / scenes] [answered]
There is old great idea of categories like Lotus Notes that is the ability to create a 'structure' similar to sub-folders.
May be it will good to categorize the automation list in HA?
Description you can read here [how to use categories to create a navigable folder structure](http://www.matnewman.com/webs/personal/matblog.nsf/dx/how-to-use-categories-to-create-a-navigable-folder-structure?opendocument&comments)

**Comment by frenck** [comment, marked as answer]
There are many of these suggestions open already.
Please use the search before opening a new discussion.

Thanks 👍

**Comment by spacegaier** [comment]
What Frenck said 👍, plus starting with the new upcoming release you can now filter automations, scenes and scripts based on area, devices, and entities which will help with identifying the automation you are searching for.
